tests/loginPage.ts

```diff
@@ -0,0 +1,25 @@
+import { Credential } from "./creditintial";
+import { Locator, Page } from "@playwright/test";
+
+export default class loginPage {
+  private pageDomain: Page;
+
+  usernameTextBox: Locator;
+  passwordTextBox: Locator;
+  loginButton: Locator;
+  invalidCredential: Locator;
+  credentialList: Credential[] = [
+    new Credential("Admin1", "passw", "fail"),
```

Comment:
why do we need to implement this way, while we are using statice values ?

Reply:
well, do you have a better idea to use diffrent credentials scenario for testing? I made here 4 scenarios for testing giving that 3 are wrong password and one correct password. what is your prototype to do it?

Reply:
since we usually use users from a secret file, so treat them as a secret and create your own secret files like this:
export const users = {
        user1: {attr1:, attr2:, ...},
user2: {},
user3: {},
}
this way and in this way and no need for this class Credential

Reply:
well, got it. please provide me with the secret file. so I learn from you

Reply:
we can't since it's a secret related to the projects we are working on so that's not as option